lib/DocumentFilter/DocumentFilterForm.test.js

```diff
@@ -148,6 +131,12 @@ describe('DocumentFilterForm', () => {
         });
       });
 
+      it('should open the document filter builder modal when the edit button is clicked', () => {
+        const { getByText } = renderComponent;
+        fireEvent.click(getByText('Edit document filters'));
```

Comment:
Why is this using fireEvent instead of interactors? 

I like the problem solving you've done behind the scenes to make sure this works, but at the bare minimum there needs to be a comment explaining why this case is different. My gut says that a difference in behaviour means either our component is wrong or the interactors are wrong... it's rare that we actually _should_ be using things outside of them.


Again, I like the implication that a bunch of work and problem solving has gone into this though, so kudos! Just worried that a "special case" when not labelled with a comment ends up becoming copied and pasted elsewhere blindly and we end up with 10 ways of doing the same thing.